### ISSUE TITLE
Flag for IPv6 not available

### ISSUE THREAD

**Issue by jbbr** [scaleway/docker-machine-driver-scaleway#50]
It would be great to be able to enable IPv6 for the created scaleway instance.

Some ipv6 code seems to be in the code but commented out:
https://github.com/scaleway/docker-machine-driver-scaleway/blob/master/driver/scaleway.go#L169

**Comment by moul** [comment]
What do you think about enabling ipv6 by default and add an option for disabling it instead ?

**Comment by jbbr** [comment]
Sounds good. But unfortunately I have some issues after enabling IPv6 for scaleway-VM's deployed with docker-machine.

Might be due to issues in the scaleway IPv6 bootstrap script or just a fault on my side - but before enabling it by default, it might be wise to test it first.
Hopefully no issues occur when IPv6 is enabled on first boot.

**Comment by jbbr** [comment]
Okay, this seems to be a bug in scaleway bootstrapping.

Enabling ipv6 from the beginning will probably work.

I will report this bug to scaleway.

Reproduce:

- Create ubuntu 16.04 docker VCS (through docker-machine create, no ipv6 by default)
- Enable IPv6 in scaleway panel and restart server
- Ipv6 connectivity is broken (routes and ips exist though)

Workaround:
- Terminate (Archive) VCS in scaleway panel
- Start VCS again
- IPv6 connectivity is working
Okay, this seems to be a bug in scaleway bootstrapping.

Enabling ipv6 from the beginning will probably work.

Reproduce:
- Create ubuntu 16.04 docker VCS (through docker-machine create, no ipv6 by default)
- Enable IPv6 in scaleway panel and restart server
- Ipv6 connectivity is broken (routes and ips exist though)

Workaround:
- Terminate (Archive) VCS in scaleway panel
- Start VCS again
- IPv6 connectivity is working

**Comment by QuentinPerez** [comment]
Hi @jbbr, thanks you for your feedback,

To enable IPV6 on VCS instances, the reboot is the normal way https://blog.scaleway.com/2016/03/31/introducing-native-ipv6-connectivity-on-scaleway/ 
`On the VPS, a reboot of the server from the console is mandatory to enable IPv6.`

**Comment by jbbr** [comment]
Hi @QuentinPerez ,
unfortunately a reboot is not sufficient. As I wrote in my reproducing steps ("Enable IPv6 in scaleway panel **and restart server**"). Tested with default VCS1 and Ubuntu Xenial setup.
I already opened a ticket for this and got a reply that they sent this to the network team.

As I don't know if others are experiencing this problem too, I would recommend to add an option for IPv6 to the scaleway driver and don't enable it by default yet.
